help/styles/styles.css

```diff
@@ -513,6 +505,85 @@ main .section:empty {
   pointer-events: none;
 }
 
+main .section.rating {
+  display: block;
+}
+
+main .section.rating .default-content-wrapper {
+  margin: unset;
+  width: unset;
+}
+
+main .section.rating .default-content-wrapper h3 {
+  font-size: 1.25rem;
+  margin-bottom: 20px;
+  font-family: var(--button-font-family);
+}
+
+main .section.rating .default-content-wrapper ul {
+  list-style-type: none;
+  padding-left: 0;
+  font-size: 1rem;
+}
+
+main .section.rating .default-content-wrapper ul li {
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+main .section.rating .default-content-wrapper p {
+  margin-top: 20px;
+  margin-bottom: 0;
+}
+
+main .section.rating .default-content-wrapper ul li a {
+  text-decoration: none;
+  color: var(--button-color);
+}
+
+main .section.rating .default-content-wrapper ul li a:hover {
+  text-decoration: underline;
+  text-decoration-color: var(--link-hover-color);
+}
+
+main .section.rating .default-content-wrapper a.button {
+  all: unset;
+  font-family: var(--button-font-family);
+  font-size: 1rem;
+  position: relative;
+  cursor: pointer;
+}
+
+main .section.rating .default-content-wrapper a.button::after {
+  content: url('../icons/chevron-black.svg');
+  margin-left: 5px;
+  position: absolute;
+  top: 1px;
+}
+
+main .section.rating .default-content-wrapper a.button:hover {
+  color: var(--link-hover-color);
+  text-decoration: underline;
+}
+
+@media (min-width: 600px) {
+  main .section.rating .default-content-wrapper {
+    margin: unset;
+    margin-left: 30%;
```

Comment:
This margin is not present in https://www.macquarie.com.au/help/personal/accessing-online-and-mobile-banking/customise-your-online-banking/customise-your-insights-page.html

Reply:
yes, I was wondering how to handle this in between transition to make it look better visually, hence I added this ...since the rating feedback part would be missing

Reply:
I made this 50% so we still see space for rating stars part which will be added later.
This will be temporary till customer actually adds the left half

Reply:
makes sense